### PR TITLE
feat: support for split data files

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Supports split `packages` data files and `scripts` data files. Also, relative paths for data files, except for `list.json`, will be available.

If apm-schema in your `node_modules` is not up to date, please run `yarn upgrade apm-schema`.

### Progress
- [x] Implementation
- [x] Preparation of test data

## Related Issue

- https://github.com/team-apm/apm-data/issues/122

This PR depends on the following PR(s)
- https://github.com/team-apm/apm/pull/532

## How Has This Been Tested?

- Can handle data added by https://github.com/team-apm/apm-data/pull/182 .

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/lib/modList.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+/**
+ * Resolve paths.
+ *
+ * @param {string} base - base path
+ * @param {string} relative - relative path
+ * @returns {string} - absolute path
+ */
+function resolvePath(base: string, relative: string) {
+  if (base.startsWith('http')) {
+    return new URL(relative, base).href;
```
Comment (hal-shu-sato) [inline src/lib/modList.ts:17]: This code may download data from external hosts or other directories, which is a security concern.

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
I know this is a draft, but I review it. Sorry.

File: src/lib/modList.ts (hunk 1 above)
Comment (hal-shu-sato) [reply src/lib/modList.ts:17]: Refer to the last example on this page: https://developer.mozilla.org/ja/docs/Web/API/URL/URL

Comment (yumetodo) [reply src/lib/modList.ts]: b6c2ea1 seems goot to me.

**Comment by mitosagi** [comment]
Rebased. No change in code.

**Review by hal-shu-sato — APPROVED** [review]
It works fine.

**Pull request merged**